src/packages/cli/utils/url-message.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+
+export default function urlMessage(): void {
+  console.log(`
+  You're using a URL in your database config (config/database.js).
+
+  In that case, Lux assumes you don't need to create or drop your database.
+  If you'd like to create or drop a database, set up your database config\
+ without the url.
+
+  For guidance, see:
+  https://github.com/postlight/lux/blob/master/src/packages/database/\
+interfaces.js#L17
```

Comment:
The message is perfect. The only thing I suggest we do is move it to a constant located in `./src/packages/cli/constants.js`.

```javascript
// ./src/packages/cli/constants.js
import template from '../../template';

export const CONNECTION_STRING_MESSAGE = template`
  You're using a URL in your database config (config/database.js).

  In that case, Lux assumes you don't need to create or drop your database.
  If you'd like to create or drop a database, set up your database config\
  without the url.

  For guidance, see:
  https://github.com/postlight/lux/blob/master/src/packages/database/\
  interfaces.js#L17
`;
```